QA.md

```diff
@@ -62,6 +66,16 @@
         1. Send 1 HTR to your own address (it will ask for the biometry again)
         1. Disable biometry
         1. Lock wallet and it must require you to type your PIN Code.
+    1. Test safe Biometry mode
+        1. Go to unleash and enable the `safe-biometry-mode.rollout` for this wallet.
+        1. Enable biometry (either Fingerprint, FaceID or TouchID depending on the device, it will require your pin)
+        1. Check that the "Change PIN" option is no longer on the list.
+        1. Send 1 HTR to your own address (it will ask for the biometry instead of pin)
+        1. Close the app and open, it should ask for biometry instead of pin when opening the wallet.
+        1. Send 1 HTR to your own address (it will ask for the biometry again)
+        1. Disable biometry (it will ask for the biometry)
+        1. Check that the "Change PIN" option appears on the list.
+        1. Lock wallet and it must require you to type your PIN Code.
```

Comment:
Does it make sense to test the migration? In this case, we would need to add one more section here with old biometry enabled, then activate in the Unleash and validate if it works fine to load the wallet and send a tx.

What do you think?